pinakes/locale/zh/LC_MESSAGES/django.po

```diff
@@ -0,0 +1,211 @@
+# SOME DESCRIPTIVE TITLE.
```

Comment:
@rooftopcellist We already have this translation file for some new added messages. Can you merge them in?

Reply:
Yep @hsong-rh , it looks like there are about 20 strings that are new in your PR, this is because you extracted strings later than I did.  Those strings will not be in the translation memory in Memsource, so I cannot guarantee that they will be the same in the next round of translations.  We should check those strings against the next round of translated strings to make sure the translators have similar translations.

Reply:
These are thee strings that were different that we should keep track of for the next round of translation - http://pastebin.test.redhat.com/1050433

Reply:
@rooftopcellist It's ok to have changes in the next round, we just use them to prove the i18n support works.